### ISSUE TITLE
Error: Generic error: The datasource db already exists in this Schema. It is not possible to create it once more.

### ISSUE THREAD

**Issue by navicstein** [prisma/migrate#616]
## Bug description

SInce i updated my prisma versions I've been having problems trying to migrate an existing schema that obviously worked

## How to reproduce
run
 ```sh
$ prisma migrate save --experimental
```
throws 

```sh
 yarn dev:migrate
yarn run v1.22.4
$ prisma migrate save --experimental  && prisma migrate up --experimental 
Environment variables loaded from prisma/.env
Prisma Schema loaded from prisma/schema.prisma
Error: Generic error: The datasource db already exists in this Schema. It is not possible to create it once more.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

## Expected behavior

show migrate successfully without errors

## Environment & setup
```sh
❯ prisma -v
Environment variables loaded from prisma/.env
@prisma/cli          : 2.9.0
@prisma/client       : 2.9.0
Current platform     : debian-openssl-1.1.x
Query Engine         : query-engine 369b3694b7edb869fad14827a33ad3f3f49bbc20 (at ../../../../../opt/node/lib/node_modules/@prisma/cli/query-engine-debian-openssl-1.1.x)
Migration Engine     : migration-engine-cli 369b3694b7edb869fad14827a33ad3f3f49bbc20 (at ../../../../../opt/node/lib/node_modules/@prisma/cli/migration-engine-debian-openssl-1.1.x)
Introspection Engine : introspection-core 369b3694b7edb869fad14827a33ad3f3f49bbc20 (at ../../../../../opt/node/lib/node_modules/@prisma/cli/introspection-engine-debian-openssl-1.1.x)
Format Binary        : prisma-fmt 369b3694b7edb869fad14827a33ad3f3f49bbc20 (at ../../../../../opt/node/lib/node_modules/@prisma/cli/prisma-fmt-debian-openssl-1.1.x)
Studio               : 0.296.0
Preview Features     : transactionApi
```

- OS:  Linux (Ubuntu 20.04)
- Database:  PostgreSQL
- Node.js version:  v14.8.0
- Prisma version: 2.9.0

**Comment by navicstein** [comment]
### Work Around
I've fixed it by deleting the `_Migration` table in my DB and `migrations` folder in Prisma directory,  but I don't just know why it stopped migrating automatically

**Comment by tomhoule** [comment]
Thanks for taking the time to report this issue!

We've recently released a Prisma Migrate Preview ([2.1.3.0 release notes](https://github.com/prisma/prisma/releases/tag/2.13.0)), which has quite a few changes compared to the previous experimental version. We believe this issue is no longer relevant in this new version, so we are closing this.

We would encourage you to try out Prisma Migrate Preview. If you come across this or any other issue in the preview version, please feel free to open a [new issue in prisma/prisma](https://github.com/prisma/prisma/issues/new/choose).
For general feedback on Prisma Migrate Preview, feel free to chime in [on this issue](https://github.com/prisma/prisma/issues/4531).
